dev/releases/make_archives.py

```diff
@@ -144,6 +145,22 @@
     # all packages, and upload that as part of the release, too; this file would be rather
     # useful for updating the website, and also for the PackageManager
     # (Why JSON? Because GAP, Python and many more can easily process it.)
+    # now create the file package-infos.json
+    # We first build the json package, then create the package-infos.json, then
+    # clean up the json package again.
+    path_to_json_package = glob.glob(f'{tmpdir}/{basename}/pkg/json*')[0]
+    with working_directory(path_to_json_package):
+        subprocess.run(["./configure"], check=True)
+        subprocess.run(["make"], check=True)
+
+    subprocess.run(["./bin/gap.sh", "-r", "--quitonbreak",
+                   "../../dev/releases/PackageInfos-to-JSON.g"],
+                   check=True)
+
+    with working_directory(path_to_json_package):
+        subprocess.run(["rm", "-rf", "bin/", "Makefile"],
```

Comment:
Perhaps also call `make clean` before that, to also remove *.o files etc.?

Reply:
Done.